src/prompts/alert/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { Component, h } from 'preact';
+import { PromptUiProps } from '../ui';
+import { AlertPromptConfig } from '../../core';
+import { PlatformConfigContext } from '../ui-context';
+
+export class Alert extends Component<PromptUiProps<AlertPromptConfig>, never> {
+    onRequestNativePrompt = () => {
+        this.props.onPromptAccepted(this.props.config);
+    };
+
+    onRequestCancel = () => {
+        this.props.onPromptDeclined(this.props.config);
+    };
+
+    render() {
+        const classes = `kumulos-prompt kumulos-prompt-${this.props.promptManagerState} kumulos-alert-container`;
+        const config = this.props.config;
+
+        const {
+            heading,
+            body,
+            declineAction,
+            acceptAction
+        } = config.labels.alert;
+        const {
+            bg,
+            fg,
+            acceptActionBg,
+            acceptActionFg,
+            declineActionBg,
+            declineActionFg
+        } = config.colors.alert;
+
+        const containerStyle = {
+            backgroundColor: bg,
+            color: fg
+        };
+
+        const declineActionStyle = {
+            backgroundColor: declineActionBg,
+            color: declineActionFg
+        };
+
+        const acceptActionStyle = {
+            backgroundColor: acceptActionBg,
+            color: acceptActionFg
+        };
+
+        return (
+            <PlatformConfigContext.Consumer>
+                {config =>
```

Comment:
Maybe more of a PITA / not super necessary to optimise this away, but should we make the render func a bound closure class member?

Thought preact had a lint rule set up about this, but perhaps in another project I'm thinking of.